spec/factories/tasks.rb

```diff
@@ -5,5 +5,10 @@
     from = Date.parse("2019/08/01")
     to   = Date.parse("2019/12/31")
     deadline { Random.rand(from..to) }
+
+    trait :status_done do
```

Comment:
ステータスがdoneになって完了日が登録されることは`done`だけでも読み取れるかと思います。
```suggestion
    trait :done do
```

Reply:
6acbd7c で修正しました。